index.php

```diff
@@ -0,0 +1,56 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <?php require_once 'php.php'; ?>
+    <link rel="stylesheet" href="modern-normalize.css">
+    <link rel="stylesheet" href="stylesheet.css">
+    <meta name = "viewpoint" content = "width = device - width, initial-scale=1.0">
+    <title>Pigment Hunting Collection App - </title>
+</head>
+
+<body>
+
+<section class ="container">
+    <table class = "table">
+        <thead>
+        <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Color</th>
+            <th>Similar HEX Code</th>
+            <th>Geology</th>
+            <th>Image Closeup</th>
+            <th>Image Site</th>
+            <th>Country</th>
+            <th>Town</th>
+            <th>Coordinates (Lat)</th>
+            <th>Coordinates (Long)</th>
+        </tr>
+        </thead>
+        <tbody>
+            <?php foreach ($collection as $pigment) :
```

Comment:
Where does `$collection` come from? It is generally bad practice to `require` in variables, since people reading the code find it hard to follow. 

In this file you should call to a function that populates your `$collection` variable. This function can be `require`d and then used in this file.